conf/battle/drops.conf

```diff
@@ -18,20 +18,23 @@ item_first_get_time: 3000
 
 // Grace time during which only the first and second person who did the most damage to a monster can get the item? (in milliseconds)
 // (Takes effect after item_first_get_time elapses)
-item_second_get_time: 1000
+item_second_get_time: 2000
 
 // Grace time during which only the first, second and third person who did the most damage to a monster can get the item? (in milliseconds)
 // (Takes effect after the item_second_get_time elapses)
-item_third_get_time: 1000
+item_third_get_time: 2000
 
-// Grace time to apply to MvP reward items when the Most Valuable Player can't get the prize item and it drops on the ground? (in milliseconds)
+// Grace time during which only the person who did the most damage to a boss can get the item? (in milliseconds)
+// Also applies to MvP reward items when the Most Valuable Player can't get the prize item and it drops on the ground
 mvp_item_first_get_time: 10000
 
-// Grace time for the first and second MvP so they can get the item? (in milliseconds)
+// Grace time during which only the first and second person who did the most damage to a boss can get the item? (in milliseconds)
+// Also applies to MvP reward items when first and second MVP can't get the prize item and it drops on the ground
 // (Takes effect after mvp_item_first_get_time elapses)
 mvp_item_second_get_time: 10000
 
-// Grace time for the first, second and third MvP so they can get the item? (in milliseconds)
+// Grace time during which only the first, second and third person who did the most damage to a boss can get the item? (in milliseconds)
+// Also applies to MvP reward items when first, second and third MvP can't get the prize item and it drops on the ground
```

Comment:
```suggestion
// Also applies to MvP reward items, when first, second and third MvP can't get the prize item and it drops on the ground
```

Reply:
This would be incorrect grammar, in English you don't put commas in front of "when" and "if".
https://www.pristineword.com/comma-when/

Reply:
I think you misunderstand your own link.
In your link it is 3. It is a parenthetic expression, that further describes the situation, in which this config applies.
It may be unnecssary, but not incorrect.

Reply:
This is not a parenthetic expression, it's clearly an independent clause.

Reply:
"it can be easily omitted without changing the meaning of the sentence"

=> It applies to MVP reward items.
The rest is just additional - too much info. Of course it can only happen if they drop to the ground, thus this can be left out and is parenthetic.